.github/workflows/deploy-screener.yml

```diff
@@ -0,0 +1,49 @@
+# This action deploys changes to the eligibility screener
+# Deploy steps:
+# - Build a docker release image and upload it to ECR and tag it.
+# - Apply the terraform changes (if applicable), pointing it to the docker image that was uploaded
```

Comment:
To clarify, this is in a different repo and is _not_ part of this workflow, right? wondering if there's a clearer way to organize this top-level comment.

Something like:

```
# This action deploys changes to the eligibility screener next.js application. 
#
# Deploy steps:
# - Build and tag a docker release image
# - Upload the docker image to ECR
# - Trigger a rotation of the ECS service containers
#
# The underlying terraform is managed in X repo and references the uploaded docker image.


```